mithril-test-lab/mithril-end-to-end/src/mithril/aggregator.rs

```diff
@@ -41,6 +41,7 @@ impl Aggregator {
                 "CARDANO_NODE_SOCKET_PATH",
                 bft_node.socket_path.to_str().unwrap(),
             ),
+            ("STORE_RETENTION_LIMIT", "5"),
```

Comment:
This is probably an option of the end to end test? I am not sure we want to prune when this is run in the CI.
Or maybe we could test with and without pruning on the CI, WDYT?

Reply:
Before I changed updated the prune to use the configuration retention limit instead of a constant it was the e2e tests that gave away that something was wrong _(the problem was that the protocol parameters were pruned but not the table that have a foreign key with it)_.
So I see value to always use enable pruning in e2e tests.
In this case I would be in favor of increasing the retention limit in order to keep most of the data while ensuring that it works without issue on 2 or 3 epochs.

Reply:
The value was increased to 10 in order to start pruning at the final third of the e2e tests (they usually go up to epoch 13~14).
Since most problems with the e2e happens in the first two thirds this will not hinders debugging.